app/components/error_text/index.js

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+import {connect} from 'react-redux';
+import {getTheme} from 'mattermost-redux/selectors/entities/preferences';
+
+import ErrorText from './error_text.js';
+
+function mapStateToProps(state, ownProps) {
+    return {
+        ...ownProps,
```

Comment:
this is not needed

Reply:
Yup. Fixed it but I amended the commit so you might have to look again